### ISSUE TITLE
Screen reader accessibility.

### ISSUE THREAD

**Issue by rkingett** [oskusalerma/trelby#424]
Upon hearing about this program, I was very eager to try it out since it is so keyboard driven. Unfortunately, NVDA, an open source screen reader, cannot read the text input area at all or certain settings dialog boxes, such as the hotkeys dialog box.

Are there plans to allow screen readers to read the text area? I am unable to review what I've typed.

I'd be happy to answer questions!
